core/templates/dev/head/dashboard/MyExplorations.js

```diff
@@ -37,11 +37,11 @@ oppia.controller('MyExplorations', [
     };
 
     $rootScope.loadingMessage = 'Loading';
-    $http.get('/myexplorationshandler/data').success(function(data) {
-      $scope.explorationsList = data.explorations_list;
+    $http.get('/myexplorationshandler/data').then(function(response) {
+      $scope.explorationsList = response.data.explorations_list;
```

Comment:
IMO, better to just do "var data = response.data;" here, and follow that same sort of standard pattern throughout the app.